masterlist.yaml

```diff
@@ -402,16 +400,10 @@ plugins:
         udr: 0
         nav: 57
     clean:
-      - crc: 0x6061bcff # v1.5.39.0 - Auto Clean Second pass
-        util: 'SSEEdit v3.2.1'
```

Comment:
Removed CRC for plugins with wild edits. Because I think marking them as clean & also having wild edits is sending mixed messages.